packages/prettier-config/prettier.config.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
```

Comment:
Sorry, but you can't just make so huge changes, this config are used by tens of Tinkoff apps (react + tramvai), and it is breaking change, without motivation

Reply:
reverted